src/web/screens/home.tsx

```diff
@@ -14,15 +16,34 @@ function Home() {
   const { LoggedInUser } = useContext(CurrentUserContext);
   const [loading, setIsLoading] = useState<boolean>(true);
   const [sharingPermissionsCount, setSharingPermissionsCount] = useState<number>(0);
+  const [bulkPermissionsCount, setBulkPermissionsCount] = useState<number>(0);
   const [hasError, setHasError] = useState<boolean>(false);
 
   useEffect(() => {
     const getSharingParticipantsCount = async () => {
       setIsLoading(true);
       try {
         const sharingList = await GetSharingList();
-        // TODO include the sites from allowed_types in the count
-        setSharingPermissionsCount(sharingList.allowed_sites.length);
+        const manualSites = sharingList.allowed_sites;
+        const allowedTypes = sharingList.allowed_types;
+        const allowedTypeMap = new Map<ClientType, boolean>();
+        allowedTypes.forEach((item) => {
+          allowedTypeMap.set(item, true);
+        });
+        const siteList = await preloadAvailableSiteList();
+        const bulkSites = siteList.filter((item) => {
+          let found = false;
+          const clientTypes = item.clientTypes || [];
+          for (let i = 0; i < clientTypes.length; i += 1) {
```

Comment:
nit: how about use `clientTypes.some(clientType => allowedTypeMap.get(clientType))` instead of for loop?

Reply:
I changed it.